reana_workflow_controller/config.py

```diff
@@ -95,6 +95,22 @@
 https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory.
 """
 
+REANA_KUBERNETES_JOBS_TIMEOUT_LIMIT = os.getenv("REANA_KUBERNETES_JOBS_TIMEOUT_LIMIT")
+"""Default timeout for user's job in seconds. Exceeding this time will terminate the job.
+
+Please see the following URL for more details
+https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-termination-and-cleanup.
+"""
+
+REANA_KUBERNETES_JOBS_MAX_USER_TIMEOUT_LIMIT = os.getenv(
+    "REANA_KUBERNETES_JOBS_MAX_USER_TIMEOUT_LIMIT"
+)
+"""Maximum custom timeout in seconds that users can assign to their job.
+
+Please see the following URL for more details
+https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-termination-and-cleanup.
+"""
+
```

Comment:
The same [comment](https://github.com/reanahub/reana/pull/610#discussion_r788621722) from the `reana` PR applies here.